analyses/sex-prediction-from-RNASeq/run-sex-prediction-from-RNASeq.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This script runs the sex-prediction-from-RNASeq analysis
+# Author's Name Bill Amadio 2019
+
+set -e
+set -o pipefail
+
+Rscript --vanilla 01-clean_split_data.R --expression ../../data/pbta-gene-expression-kallisto.stranded.rds --metadata ../../data/pbta-histologies.tsv --output_directory processed_data --filename_lead kallisto_stranded
+
+Rscript --vanilla 02-train_elasticnet.R --input_directory processed_data --output_directory models --filename_lead kallisto_stranded
+
+Rscript --vanilla 03-evaluate_model.R --test_set_input_directory processed_data --model_input_directory models --output_directory results --filename_lead kallisto_stranded
```

Comment:
Can you add running this again with `--target_column germline_sex_estimate` and commit the results? So I'm proposing two `03-evaluate_model.R` steps.

Reply:
I pushed changes that

1) fix script directory issue
2) eliminate all file name constructions by paste
3) run 03-evaluate_model twice, once for reported_gender and once for germline_sex_estimate
4) track both models and results directories

git did not record changes to the file kallisto_stranded_36354_0.25_model_transcripts.RDS in the models directory.  The file time stamp (today) is the same as the other two files in the models directory, and I am sure it is new since I delete all output files and directories before a test run of the scripts.